tests/unit/lib/WUI/nhttp/time_mock.cpp

```diff
@@ -0,0 +1,18 @@
+#include "time_mock.h"
+
+namespace {
+
+uint32_t mock_time = 0;
+
+}
+
+void set_time(uint32_t time) {
+    mock_time = time;
+}
```

Comment:
Are we using this one in here?

Reply:
Yes, we are using it in the server tests to simulate stale nonce.